script.js

```diff
@@ -4,18 +4,49 @@ window.addEventListener("DOMContentLoaded", function () {
     console.log("%c box", "color: #e57373", box);
   };
 
-  new SwipeableBottomSheet({ triggerId: "trigger-1", bottomSheetId: "swipeable-bottom-sheet-1", onOpen: handleOnOpen });
-  new SwipeableBottomSheet({ triggerId: "trigger-2", bottomSheetId: "swipeable-bottom-sheet-2" });
-  new SwipeableBottomSheet({ triggerId: "trigger-3", bottomSheetId: "swipeable-bottom-sheet-1", overlay: false });
-  new SwipeableBottomSheet({ triggerId: "trigger-10", bottomSheetId: "swipeable-bottom-sheet-2", overlay: false });
-  new SwipeableBottomSheet({ triggerId: "trigger-4", bottomSheetId: "swipeable-bottom-sheet-2", peek: "50vh" });
+  new SwipeableBottomSheet({
+    triggerId: "trigger-1",
+    bottomSheetId: "swipeable-bottom-sheet-1",
+    onOpen: handleOnOpen,
+  });
+  new SwipeableBottomSheet({
```

Comment:
Sorry about this formatting, vscode auto formatted it.

Reply:
No worries, I have to add this tooling & build setup with Parcel anyways
https://github.com/timelessco/scripbox-mf

I will do it this weekend.